src/autotype/AutoTypeSelectDialog.h

```diff
@@ -63,6 +65,7 @@ private slots:
 
     QList<QSharedPointer<Database>> m_dbs;
     QList<AutoTypeMatch> m_matches;
+    AutoTypeMatch m_lastMatch;
```

Comment:
This only seems to be used in 1 function, recommend not storing it.

Reply:
There are multiple call sites for `performSearch()` and it's always called from within the class so passing it as an argument wouldn't work, what would you suggest?